src/rgw/rgw_op.cc

```diff
@@ -702,7 +702,7 @@ int rgw_build_bucket_policies(rgw::sal::RGWRadosStore* store, struct req_state*
  * Returns: 0 on success, -ERR# otherwise.
  */
 int rgw_build_object_policies(rgw::sal::RGWRadosStore *store, struct req_state *s,
-			      bool prefetch_data)
+			      bool prefetch_data, prefetch_range *range)
```

Comment:
nit: can we call this variable `prefetch` instead of `range` everwhere? conceptually, it's the prefetching behavior that is optional, not just the ability to specify a range